v5/tutorials/topical/multitasking.rst

```diff
@@ -55,11 +54,11 @@ Tasks in PROS are simple to create:
            :linenos:
 
             void my_task_fn(void* param) {
-                printf("Hello %s\n", (char*)param);
+                printf("Function Parameters: %s\n", (char*)param);
                 // ...
             }
             void initialize() {
-                task_t my_task = task_create(my_task_fn, "PROS", TASK_PRIORITY_DEFAULT,
+                task_t my_task = task_create(my_task_fn, "parameter(s) here", TASK_PRIORITY_DEFAULT,

```

Comment:
dont even bother passing parameters in this example. you explain how to do that in the next one